test/special/assume.c

```diff
@@ -0,0 +1,12 @@
+#include "smack.h"
```

Comment:
You should add these new regressions into our travis CI script. That should be easy to do.

Reply:
Updated .travis.yml.